app/auth/views.py

```diff
@@ -0,0 +1,56 @@
+from flask import render_template, redirect, request, url_for, flash, Flask, abort ,g
+from flask.ext.login import login_user, logout_user, login_required, current_user
+from . import auth
+from .. import db
+from .forms import LoginForm
```

Comment:
@marianodo I get an error here, maybe you forgot to upload the `app/auth/forms.py` file?